documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_idShortPath.adoc

```diff
@@ -7,14 +7,14 @@ https://creativecommons.org/licenses/by/4.0/).
 SPDX-License-Identifier: CC-BY-4.0
 
 Illustrations:
-Plattform Industrie 4.0; Anna Salari, Publik. Agentur f�r Kommunikation GmbH, designed by Publik. Agentur f�r Kommunikation GmbH
+Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
 ////
 
 === Format "Path" (IdShortPath Serialization) in JSON
 
-To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms of an idShortPath notation to be returned in an unnamed JSON-array. The notation differs depending on whether a SubmodelElementCollection or a SubmodelElementList is used. In the first case, the submodel element's idShort is separated by "." (dot) from top level down to child level. In the second case, square brackets with an index "[\<<index>>]" are appended after the idShort of the containing SubmodelElementList.
+To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms of an idShortPath notation to be returned in an unnamed JSON-array. The notation differs depending on whether a SubmodelElementCollection or a SubmodelElementList is used. In the first case, the submodel element's idShort is separated by "." (dot) from top level down to child level. In the second case, square brackets with an index "[\<<index>>]" are appended after the idShort of the containing SubmodelElementList. In any case, the first item of any idShortPath is the idShort if the requested element.
 
-In the following example, where a request for idShort paths starts at _MySubmodelElementCollection_ with SerializationModifier level = deep, the list of idShort paths is returned as follows:
+In the following example, where a request for idShort paths targets a  _MySubmodelElementCollection_ with SerializationModifier level = deep, the list of idShort paths is returned as follows:
```

Comment:
Why target? It is the starting point (relative to this collection)

Reply:
In TF API we agreed on terminology "requesting element" (this is the first in the list)